src/main.ts

```diff
@@ -97,31 +107,33 @@ wss.on("connection", (ws: WSExtended) => {
           answers: [],
         };
 
-        ws.send(formatJson({ success: true, message: "Room hosted" }));
+        ws.send(
+          formatJson({ success: true, message: SuccessMessages.RoomCreated })
+        );
         break;
 
       case RequestType.REGISTER_USER:
         if (!checkAttendee(request.designation))
           return ws.send(
             formatJson({
               error: true,
-              message: "Designation does not support registration",
+              message: ErrorMessages.WrongDesignationRegistration,
             })
           );
 
         if (!roomExists(Object.keys(state.rooms), request.roomId))

```

Comment:
Logic error in room checking, should return if room exists, but is returning on if room doesn't exist